### PR TITLE
[release 7] Update changelog from release notes.

### PR DESCRIPTION
**Pull request by chriselion**
### Proposed change(s)
Add back a few of the larger changes from the release notes.

### Types of change(s)
- [x] Documentation update

### Checklist
- [x] Updated the [documentation](https://github.com/Unity-Technologies/ml-agents/tree/master/docs) (if applicable)

### CODE REVIEW

File: com.unity.ml-agents/CHANGELOG.md [hunk 1]
```diff
@@ -13,9 +13,15 @@ and this project adheres to
 
 ### Minor Changes
 #### com.unity.ml-agents (C#)
+- The `IActuator` interface and `ActuatorComponent` abstract class were added.
+These are analogous to `ISensor` and `SensorComponent`, but for applying actions
+for an Agent. They allow you to control the action space of an Agent more
+programmatically than defining the actions in the `BrainParameters`. See
```
Comment (hvpeteet) [inline com.unity.ml-agents/CHANGELOG.md:19]: Is this the old name of `Behavior Parameters`? If so can you update it to match?

**Review by hvpeteet — APPROVED** [review]
Thanks Chris.

I went ahead and approved but left a comment on one piece that confused me. I think it is just an outdated term but I wanted to double check.

File: com.unity.ml-agents/CHANGELOG.md (hunk 1 above)
Comment (chriselion) [reply com.unity.ml-agents/CHANGELOG.md:19]: The action space sizes are technically part of the BrainParameters class
https://github.com/Unity-Technologies/ml-agents/blob/4845314ade08d9456570cc520869c04c5df78a51/com.unity.ml-agents/Runtime/Policies/BrainParameters.cs#L64-L76
although they appear in the "Behavior Parameters" inspector window. I'll reword to be more generic.

Comment (hvpeteet) [reply com.unity.ml-agents/CHANGELOG.md]: Thanks

**Pull request merged**